pages/cart/cart.css

```diff
@@ -0,0 +1,37 @@
+.cart-page{
+    display:flex;
+    justify-content:space-around;
+    align-items: flex-start;
+    margin: 2rem 0;
+    width:100%;
+    padding:0 2rem;
+    gap:2rem;
+}
+.cart-sec {
+    flex-grow:1;
+  padding: 2rem 5%;
```

Comment:
I'm not sure if this is good practice; unless you have a particular purpose here, stick to consistent units.